MetaMask/metamask-mobile
# fix: QR scanner crash when user dismisses camera view in web view

## **Description**

Added try catch block to stop uncaught error crashing the app when user dismisses the QR scanner in the webview.
App only crashes when the user presses the close button at the top right corner of the screen.

## **Related issues**

Fixes: #4848 

## **Manual testing steps**

- Must be tested on physical device. Will not work on iOS simulator.

1. Open the in app web browser and go to: https://pkeshkamat.github.io/poc2.html
2. On the QR scanning screen, press the close button on the top right corner of the screen
3. Observe that the app does not crash and instead goes back to the main app view

## **Screenshots/Recordings**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
